dev-python/pytest-randomly/pytest-randomly-3.12.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Pytest plugin to randomly order tests and control random.seed"
+HOMEPAGE="
+	https://github.com/pytest-dev/pytest-randomly/
+	https://pypi.org/project/pytest-randomly/
+"
+SRC_URI="$(pypi_sdist_url --no-normalize)"
```

Comment:
`PYPI_NO_NORMALIZE=1` above `PYTHON_COMPAT` and then you don't have to override vars.